includes/class-wc-payments-incentives-service.php

```diff
@@ -41,6 +41,7 @@ public function init_hooks() {
 		add_action( 'admin_menu', [ $this, 'add_payments_menu_badge' ] );
 		add_filter( 'woocommerce_admin_allowed_promo_notes', [ $this, 'allowed_promo_notes' ] );
 		add_filter( 'woocommerce_admin_woopayments_onboarding_task_badge', [ $this, 'onboarding_task_badge' ] );
+		add_filter( 'woocommerce_admin_woopayments_onboarding_task_additional_data', [ $this, 'onboarding_task_additional_data' ], 20 );
```

Comment:
Used a `20` priority instead of the default `10` to make sure we overwrite any previously added incentives additional data.

Reply:
If these changes are active with a WooCommerce version that doesn't fire the `woocommerce_admin_woopayments_onboarding_task_additional_data` filter hook, no harm done. The `onboarding_task_additional_data` method will simply not be called.